StackExchange.Redis.Tests/Performance.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace StackExchange.Redis.Tests
 {
+    [Collection(NonParallelCollection.Name)]
```

Comment:
This isn't related to the PR, just fixing the flakiness of this test in general.